doc/rtd/development/first_PR.rst

```diff
@@ -14,37 +14,14 @@ Sign the CLA
 ============
 
 To contribute to cloud-init, you must first sign the Canonical
-`contributor license agreement <CLA_>`_ (CLA).
-
-If you have already signed it as an individual, your Launchpad username will be
-listed in the `contributor-agreement-canonical`_ group. Unfortunately there is
-no easy way to check if the organisation or company you are working for has
-signed it.
-
-When you sign:
-
-* ensure that you fill in the GitHub username field,
-* when prompted for a 'Project contact' or 'Canonical Project Manager', enter
-  'James Falcon'.
-
-If your company has signed the CLA for you, please contact us to help in
-verifying which Launchpad/GitHub accounts are associated with the company.
+`contributor license agreement <CLA_>`_ (CLA). A check is run against
+every pull request to ensure that the CLA has been signed.
```

Comment:
@TheRealFalcon Shall we make a note that August 2025 may require renewing your CLA signing?

Reply:
I don't think that makes sense in our first PR documentation as the people signing now won't need to resign. I think we'll just need to address it individually as it comes up on a PR.